dascore/io/segy/core.py

```
@@ -87,6 +87,11 @@ def write(self, spool: dc.Patch | dc.BaseSpool, resource, **kwargs):
         _write_segy(spool, resource, self.version, segyio)
 
 
+class SegyV0_0(SegyV1_0):  # noqa
+    """An IO class supporting version 0.0 of the SEGY format. Or if the version is not set. """
```

Ruff (0.8.2)

No whitespaces allowed surrounding docstring text

Trim surrounding whitespace

(D210)